cockatrice/src/tab_deck_editor.cpp

```diff
@@ -988,6 +995,13 @@ void TabDeckEditor::actRemoveCard()
     if (!currentIndex.isValid() || deckModel->hasChildren(currentIndex))
         return;
     deckModel->removeRow(currentIndex.row(), currentIndex.parent());
+
+    DeckLoader *const deck = deckModel->getDeckList();
+    if (deck->isEmpty()) {
```

Comment:
Can simplify to `setSaveStatus(!deck->isEmpty())`

Reply:
Of course! I'll commit that ASAP